content/en/blog/falco-aks-audit-logs-plugin/index.md

```diff
@@ -0,0 +1,124 @@
+2024/12/17 21:02:07 [k8saudit-aks] opened blob checkpoint connection
+2024/12/17 21:02:07 [k8saudit-aks] opened consumer client
+2024/12/17 21:02:07 [k8saudit-aks] created eventhub processor
+```
+
+## Testing out!
+
+Create **falco_aks_audit.yaml** to test events:
+```yaml
+- rule: K8s Audit Event Detected
+  desc: A test rule that detects any Kubernetes audit event
+  condition: ka.req exists
+  output: "K8s Audit Event Detected: %ka.req"
+  priority: DEBUG
+  source: k8s_audit
+  tags: [testing, k8s_audit]
+```
+
+Load it and start falco:
+```yaml
+rules_files:
+  - /etc/falco/falco_rules.yaml
+  - /etc/falco/falco_rules.local.yaml
+  - /etc/falco/falco_aks_audit.yaml
+
+priority: debug
+```
+
+```bash
+$ falco -c /etc/falco/falco.yaml
+```
+
+Then, you should see initialization message, followed by some events from your AKS cluster. Since we have debug enabled, you should see some events from the aksService:
+```
+Thu Dec 19 11:44:55 2024: Falco version: 0.39.2 (aarch64)
+Thu Dec 19 11:44:55 2024: Falco initialized with configuration files:
+Thu Dec 19 11:44:55 2024:    /etc/falco/config.d/engine-kind-falcoctl.yaml | schema validation: ok
+Thu Dec 19 11:44:55 2024:    /etc/falco/falco.yaml | schema validation: ok
+Thu Dec 19 11:44:55 2024: System info: Linux version 6.8.0-51-generic (buildd@bos03-arm64-031) (aarch64-linux-gnu-gcc-13 (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0, GNU ld (GNU Binutils for Ubuntu) 2.42) #52-Ubuntu SMP PREEMPT_DYNAMIC Thu Dec  5 13:32:09 UTC 2024
+Thu Dec 19 11:44:55 2024: Loading plugin 'k8saudit-aks' from file /usr/share/falco/plugins/libk8saudit-aks.so
+Thu Dec 19 11:44:55 2024: Loading plugin 'json' from file /usr/share/falco/plugins/libjson.so
+Thu Dec 19 11:44:55 2024: Loading rules from:
+Thu Dec 19 11:44:55 2024:    /etc/falco/falco_rules.yaml | schema validation: ok
+Thu Dec 19 11:44:55 2024:    /etc/falco/falco_rules.local.yaml | schema validation: none
+Thu Dec 19 11:44:55 2024:    /etc/falco/falco_aks_audit.yaml | schema validation: ok
+Thu Dec 19 11:44:55 2024: The chosen syscall buffer dimension is: 8388608 bytes (8 MBs)
+Thu Dec 19 11:44:55 2024: Starting health webserver with threadiness 4, listening on 0.0.0.0:8765
+Thu Dec 19 11:44:55 2024: Loaded event sources: syscall, k8s_audit
+Thu Dec 19 11:44:55 2024: Enabled event sources: k8s_audit, syscall
+Thu Dec 19 11:44:55 2024: Opening 'k8s_audit' source with plugin 'k8saudit-aks'
+2024/12/19 14:44:55 [k8saudit-aks] opened connection to blob storage
+2024/12/19 14:44:55 [k8saudit-aks] opened blob checkpoint connection
+2024/12/19 14:44:55 [k8saudit-aks] opened consumer client
+2024/12/19 14:44:55 [k8saudit-aks] created eventhub processor
+Thu Dec 19 11:44:55 2024: Opening 'syscall' source with modern BPF probe.
+Thu Dec 19 11:44:55 2024: One ring buffer every '2' CPUs.
+```
+
+```yaml
+10:52:03.348668000: Debug K8s Audit Event Detected: verb=create, user=aksService, groups=(system:masters,system:authenticated), target=<NA>
+```
```

Comment:
Can we append a [Let's meet section](https://falco.org/blog/falco-talon-v0-3-0/#let-s-meet) (or something like that) at the bottom?

The goal is to invite users to join the community.